src/views/settings/index.tsx

```diff
@@ -1,12 +1,62 @@
 import { Link } from "react-router-dom";
+import useStore from "../../hooks/useStore";
+import { Button, createStyles, ScrollArea, TextInput } from "@mantine/core";
+import { open as dialogOpen } from "@tauri-apps/api/dialog";
+
+const useStyles = createStyles((theme) => ({
```

Comment:
Actually, you can pass just the object instead of a function, that's even better.
```ts
const useStyles = createStyles({/* Style goes here */});
```